src/main/java/codeu/controller/ProfilePageServlet.java

```diff
@@ -0,0 +1,74 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.util.UUID;
+import java.time.Instant;
+
+/**
+ * Servlet class responsible for user registration.
+ */
+public class ProfilePageServlet extends HttpServlet {
+
+  /**
+   * Store class that gives access to Users.
+   */
+  private UserStore userStore;
```

Comment:
Let's hold off on adding this UserStore (and its associated init() and setUserStore()) until after we're actually implementing storage for profiles. When you do this, there will also be a bunch of import statements that you can remove because they'll no longer be used.